api/admin/controller/catalog_services.py

```diff
@@ -20,6 +23,9 @@ def __init__(self, manager):
         super(CatalogServicesController, self).__init__(manager)
         service_apis = [MARCExporter]
         self.protocols = self._get_integration_protocols(service_apis, protocol_name_attr="NAME")
+        self.protocols[0]['settings'].append(
+            MARCExporter.get_storage_settings(self._db)
+        )
```

Comment:
I'm suspicious any time `[0]` shows up in non-test code but I checked and this is okay -- you put in one API and you're guaranteed to get one dictionary back.